composer.json

```diff
@@ -19,6 +19,7 @@
     "require": {
         "yiisoft/yii2-codeception": "dev-master",
         "yiisoft/yii2-gii": "dev-master",
-        "yiisoft/yii2-debug": "dev-master"
+        "yiisoft/yii2-debug": "dev-master",
+        "creocoder/yii2-flysystem": "0.8.*"
```

Comment:
Also, why dotplant-dev repo? Put it to the main https://github.com/DevGroup-ru/dotplant
Flysystem is needed not only for developers